clickhouse_driver/log.py

```diff
@@ -31,8 +31,9 @@ def log_block(block):
         thread_id = row.get('thread_id') or row['thread_number']
 
         logger.info(
-            '{%s} [ %s ] <%s> %s: %s',
+            '{%s} [ %s ] [ %s ] <%s> %s: %s',
             row['query_id'],
+            row['host_name'],
```

Comment:
Good. But it seems that `host_name` [should be before](https://github.com/ClickHouse/ClickHouse/blob/13d22bcd422f65141a5ddea7a75d1bd332b054c5/src/DataStreams/InternalTextLogsRowOutputStream.cpp#L37) `query_id`.

Reply:
So as thread_id